spec/controllers/session_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'spec_helper'
+
+describe SessionsController do
+  describe "GET new" do
+    before do
+      get :new
+    end
+
+    it "redirects to show videos page if logged in" do
+      user = Fabricate(:user)
+      cookies.signed[:auth_token] = user.auth_token
+      get :new
+      expect(response).to redirect_to(videos_path)
+    end
+    
+    it "renders new template if not logged in" do
+      expect(response).to render_template(:new)
+    end
+  end
+
+  describe "POST create" do
+    let(:user) {Fabricate(:user)}
+
+    context "user authenticates" do
+      before do
+        post :create, email: user.email, password: user.password, remember_me: "1"
+      end
+
+      it "displays successful flash message" do
+        expect(flash["success"]).not_to be_nil
+      end
+
+      it "should assign a permanent cookie if remember me selected" do
+        expect(cookies.signed[:auth_token]).to eq(user.auth_token)
+      end
+
+      it "should redirect to video show page" do
+        expect(response).to redirect_to(videos_path)
```

Comment:
Your spec description is "should redirect to video show page" yet you are asserting that the redirect is to the videos index. The videos index and the video show page are two different things.